### PR TITLE
Merge_Fuel_IT: enable pompe bianche

### PR DESCRIPTION
**Pull request by fansanelli**
"Pompe Bianche" are fuel station without a brand and have been skipped since the analyzer got created... This leads to confusion due to warning about unknown or missing ref:mise, let's allow them now.

Thanks
Francesco

### CODE REVIEW

File: analysers/analyser_merge_fuel_IT.py [hunk 1]
```diff
@@ -71,7 +71,7 @@ def __init__(self, config, logger = None):
                     mapping1 = {
                         'ref:mise': 'idImpianto',
                         'operator': lambda res: italian_strings.normalize_common(res['Gestore']),
-                        'brand': 'Bandiera',
+                        'brand': lambda res: res['Bandiera'] if res['Bandiera'] != 'Pompe Bianche' else Mapping.delete_tag,
```
Comment (frodrigo) [inline analysers/analyser_merge_fuel_IT.py:74]: Since you exclude the data at load, you does not need this change.

Comment (fansanelli) [reply analysers/analyser_merge_fuel_IT.py:74]: I need it in case of rebrand

Comment (frodrigo) [reply analysers/analyser_merge_fuel_IT.py:74]: No. `res` is only OpenData. In this code you does not have access to OSM data.

Comment (fansanelli) [reply analysers/analyser_merge_fuel_IT.py:74]: I've just tested it and it adds the row to remove the brand in the XML as expected for pompe bianche.. what do you mean?

Comment (frodrigo) [reply analysers/analyser_merge_fuel_IT.py:74]: At the load step you exclude row with 'Pompe Bianche'. So in the folowing step on data process you have no more this data, and so the `if res['Bandiera'] != 'Pompe Bianche' ` is useless.

It you really want to suggest delete of OSM tags brand=Pompe Bianche, keep the Mapping.delete_tag, but you have to include the the row with ''Pompe Bianche' on load step.

Comment (fansanelli) [reply analysers/analyser_merge_fuel_IT.py:74]: Check row 61

Comment (frodrigo) [reply analysers/analyser_merge_fuel_IT.py:74]: Ho, sorry, I read the diff in opposite way!
So, it s good for me.

Comment (fansanelli) [reply analysers/analyser_merge_fuel_IT.py:74]: Nevermind and thanks

**Pull request merged**